l10n_it_fatturapa_export_zip/wizard/export_fatturapa.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+import base64
+import io
+import zipfile
+from datetime import datetime
+from odoo import models, api, fields, _
+from odoo.exceptions import UserError
+
+
+class WizardAccountInvoiceExport(models.TransientModel):
+    _name = "wizard.fatturapa.export"
+
+    @api.model
+    def _default_name(self):
+        return "%s_%s" % (
+            _("E-invoice-export"), datetime.now().strftime('%Y%m%d%H%M'))
+
+    data = fields.Binary("File", readonly=True)
+    name = fields.Char('Filename', default=_default_name)
+
+    @api.multi
+    def export_zip(self):
+        attachments = []
+        for att in self.env[self._context['active_model']].browse(
+                self._context['active_ids']):
+            attachments += [att]
+        for att in attachments:
+            if att.exported_zip:
+                raise UserError(_(
+                    "Attachment %s already exported. Remove ZIP file first"
+                ) % att.name)
```

Comment:
```suggestion
                ) % att.display_name)
```
When showing records to the user, I think `display_name` should be used.
Same for other exceptions